engines/kyra/graphics/screen_eob.cpp

```diff
@@ -284,7 +326,75 @@ void Screen_EoB::loadShapeSetBitmap(const char *file, int tempPage, int destPage
 	_curPage = 2;
 }
 
+void Screen_EoB::eob2ChineseLZUncompress(byte *dest, uint32 destSize, Common::SeekableReadStream *src) {
+	src->skip(6);
+	eob2ChineseLZInStream in(src);
+	int lzOffset, lzLen;
+	byte *destPtr = dest;
+	byte *destEnd = dest + destSize;
+
+	while (1) {
+		if (in.getBit()) {
+			assert(destPtr < destEnd);
+			*destPtr++ = in.getByte();
+			continue;
+		}
+
+		if (!in.getBit()) {
+			lzLen = in.getBit() << 1;
+			lzLen |= in.getBit();
+			lzOffset = 0x100 - in.getByte();
```

Comment:
MSVC throws an error here due to getBit() returning a bool.

Reply:
Fixed